### PR TITLE
Update developer with facebook

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -224,6 +230,7 @@ websites:
     - name: Papertrail
       url: https://papertrailapp.com/
       twitter: papertrailapp
+      facebook: papertrail.io
```
Comment (fpigerre) [inline _data/developer.yml]: It looks like papertrailapp.com and papertrail.io might be separate companies. Their websites seem quite different. They also have different logos.

Comment (Jawshy) [reply _data/developer.yml]: I concur. The Facebook page for [papertrailapp.com](https://papertrailapp.com/) appears to be @[papertrailapp](https://www.facebook.com/papertrailapp/), but it is not verified and has virtually no activity.

File: _data/developer.yml [hunk 2]
```diff
@@ -352,5 +365,6 @@ websites:
     - name: Zapier
       url: https://www.zapier.com
```
Comment (Jawshy) [inline _data/developer.yml]: Their current URL is now `https://zapier.com/`. Could you please update the `url` to reflect this?

File: _data/developer.yml [hunk 3]
```diff
@@ -310,12 +320,14 @@ websites:
...
       url: https://www.statuspage.io
       twitter: statuspageio
+      facebook: statuspageio
       img: statuspageio.png
       tfa: No
 
     - name: Taiga.io
       url: https://taiga.io
       twitter: taigaio
+      facebook: taigaio
```
Comment (Jawshy) [inline _data/developer.yml:330]: Their last post appears to be from November 2015 and they do not post often. Should this still be added?

File: _data/developer.yml [hunk 4]
```diff
@@ -291,6 +300,7 @@ websites:
     - name: SourceForge
       url: http://sourceforge.net/
```
Comment (Jawshy) [inline _data/developer.yml]: They are now using HTTPS. Could you please update the `url`?

File: _data/developer.yml [hunk 5]
```diff
@@ -114,6 +118,7 @@ websites:
     - name: Docker
       url: https://docker.com/
```
Comment (Jawshy) [inline _data/developer.yml]: Their current URL is now `https://www.docker.com/`. Could you please update the `url` to reflect this?

File: _data/developer.yml [hunk 6]
```diff
@@ -94,6 +97,7 @@ websites:
     - name: Codeship
       url: https://codeship.io/
```
Comment (Jawshy) [inline _data/developer.yml]: I am getting `NET::ERR_CERT_COMMON_NAME_INVALID` for this URL. `https://codeship.com/` works fine. Perhaps use the latter instead?

Comment (stephengroat) [reply _data/developer.yml]: :heavy_check_mark:

File: _data/developer.yml (hunk 5 above)
Comment (stephengroat) [reply _data/developer.yml]: :heavy_check_mark:

File: _data/developer.yml (hunk 4 above)
Comment (stephengroat) [reply _data/developer.yml]: :heavy_check_mark:

File: _data/developer.yml (hunk 3 above)
Comment (stephengroat) [reply _data/developer.yml:330]: why not? same possibility with emails, at least facebook is public shaming

File: _data/developer.yml (hunk 2 above)
Comment (stephengroat) [reply _data/developer.yml]: :heavy_check_mark:

File: _data/developer.yml (hunk 3 above)
Comment (Jawshy) [reply _data/developer.yml:330]: Indeed. 

_adds label_

**Pull request merged**